x/escrow/init_test.go

```diff
@@ -30,7 +30,7 @@ func TestGenesisKey(t *testing.T) {
           "whole": 123456789
         }
       ],
-      "arbiter": "foo/bar/636f6e646974696f6e64617461",
+      "arbiter": "0000000000000000000000000000000000000001",
```

Comment:
This is an odd address.
WHy not a more realistic one, like recipient

Reply:
Or better yet, keep the old condition and make it an address:

`cond:foo/bar/636f6e646974696f6e64617461`

I think we need to document our features much better if our own devs don't use them